docs/content/concepts/partitions-schedules-sensors/partitions.mdx

```diff
@@ -5,19 +5,19 @@ description: Partitioned assets and jobs enable launching backfills, where each
 
 # Partitioned assets and jobs
 
-Partitioning allows a software-defined asset or job to correspond to a set of entities with identical structure but different parameters.
+_Partitioning_ allows a software-defined asset to represent a collection of partitions that can be tracked and materialized independently. In many ways, each partition functions like its own mini-asset, but they all share a common structure and dependencies. Typically, each partition will correspond to a separate file, or a slice of a table in a database.
 
-A _partitioned asset_ is an asset that's composed of a set of partitions, which can be materialized and tracked independently. Most commonly, each partition represents all the records in a data set that fall within a particular time window. Depending on where the asset is stored, each partition might correspond to a file or a slice of a table in a database.
+A common use is for each partition to represent all the records in a data set that fall within a particular time window, e.g. hourly, daily or monthly. Alternatively, each partition can represent a region, a customer, an experiment - any entity that you want to be able to materialize and monitor independently.
```

Comment:
Nit: You do a great job of giving examples for discrete types of partitions, but entity is a bit of a catch-all term. Thoughts on something more specific, like "category" or "dimension"?

Reply:
Will change to "dimension"

Reply:
Hmm but you can't materialize a "dimension" or "category"-- these are concepts for organizing the things that you _can_ materialize.

Reply:
Have you looked a the new wording? I tried to address that.

Reply:
nice ✅